### ISSUE TITLE
The availability of fault localization results of Flacoco

### ISSUE THREAD

**Issue by DehengYang** [SpoonLabs/flacoco#146]
Dear Sirs, 
Sorry for interrupting with this issue. I would like to ask that if the Flacoco localization results (e.g., the ranking list of suspicious statements) of Defects4J bugs could be released to ease the research. In this way, the results could be directly used by researchers or other potential users to avoid the possible implementation bias or inconsistent configuration bias.  

It would be sincerely appreciated if any such data could be provided publicly online or shared via email. Thank you in advance for your time and help.

**Comment by monperrus** [comment]
yes, it's a good idea. something like one CSV file per Defects4J bug?

**Comment by DehengYang** [comment]
I think it would be perfect if the complete results of flacoco could be shared, including the raw spectrum and the calculated ranking list of suspicious statements. If this is not plausible due to, for example, the large size of the spectrum, it would also help by sharing the ranking list of each bug (e.g., the results available at https://github.com/TruX-DTF/FL-VS-APR/tree/master/FaultLocalization).
